spec/function.dd

```diff
@@ -357,7 +357,13 @@ $(GNAME OutStatement):
 
 $(H2 $(LNAME2 function-return-values, Function Return Values))
 
-        $(P Function return values not marked as $(D ref) are considered to be rvalues.
+        $(P At least one $(DDSUBLINK spec/statement, return-statement, return statement),
+        throw statement, noreturn statement or assert(0) expression
```

Comment:
Link to noreturn statement docs?

Reply:
I've now added links and expanded on 'noreturn statement' which isn't really a thing.